### DISCUSSION TITLE
How to pass image from fs to rust to JS?

### DISCUSSION THREAD

**Discussion by linonetwo** [tauri-apps/tauri#2491] [Q&A]
I have a game written in PIXI.js, I hope I can load images via API exposed from the Rust side, and use Rust to read images from fs. Is it possible to pass png images using exposed API?

**Comment by tcastelly** [comment]
Yes, you can load images with the API and use Rust to access to your file system to read files.
Here an example:
https://github.com/tauri-apps/tauri/blob/dev/examples/api/src/components/FileSystem.svelte#L42
